packages/editor-ui/src/Interface.ts

```diff
@@ -168,6 +169,7 @@ export interface INodeUpdatePropertiesInformation {
 export type XYPosition = [number, number];
 
 export interface INodeUi extends INode {
+	[x: string]: unknown;
```

Comment:
I don't remember adding that and It looks like it was added on this branch so I guess it was something added accidentally. Just searched the code, couldn't find any references to it so will be removing it since it seems to work fine without it.